build/webpack/webpack.config.base.js

```diff
@@ -64,7 +110,7 @@ module.exports = ({
       __filename: false,
       // We provide our own "timers" import above, any usage of setImmediate inside
       // one of our renderer bundles should import it from the 'timers' package
-      setImmediate: false,
+      setImmediate: false
```

Comment:
Trailing commas are actually pretty useful. They make diffs and `git blame` cleaner.